pkg/data/data.go

```diff
@@ -12,8 +12,30 @@ import (
 // since stat.MaxLife is returning max life without stats, we are setting the max life value that we read from the
 // game memory, overwriting this value each time it increases. It's not a good solution but it will provide
 // more accurate values for the life %. This value is checked for each memory iteration.
-var maxLife = 0
-var maxLifeBO = 0
+type PointCounter struct {
+	MaxPoint   int
+	MaxPointBo int
+}
+
+func (pc PointCounter) Percent(point int, maxPoint int, hasBo bool) int {
```

Comment:
```suggestion
func (pc *PointCounter) Percent(point int, maxPoint int, hasBo bool) int {
```

I think we need to make the receiver a pointer, otherwise will not modify the maxHp/maxMp structures

Reply:
Yes, that is not correct~